### PR TITLE
Add api docs

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/encoding-down/issues/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+```js
+var levelup = require('levelup')
+var leveldown = require('leveldown')
+var enc = require('encoding-down')
+const db = levelup(enc(leveldown('./db')))
+db.put('name', 'encoding-down', function (err) {
+  db.get('name', function (err, value) {
+    if (!err) console.log('name=', value)
+  })
+})
```
Comment (juliangruber) [inline README.md]: y u mix var and const? :P

Comment (ralphtheninja) [inline README.md]: Aah my bad. First I wrote only const then replaced with var. Wdyt? I don't really care which as long as it's consistent

Comment (juliangruber) [inline README.md]: agreed, i favor the new const/let stuff though

Comment (ralphtheninja) [inline README.md]: Lets make it so!

**Comment by ralphtheninja** [comment]
> you rock!

It's all based on your work @juliangruber, just trying to make something each day ;)

**Comment by ralphtheninja** [comment]
I want to start working on tests next, but need a bit more focus to do that. I'm slightly drunk and out in the bush :grinning:

**Pull request merged**